lib/libyui_client/app.rb

```diff
@@ -38,17 +48,27 @@ def checkbox(filter)
     # @param filter [Hash] filter to find a widget
     # @return [Widgets::Combobox] new instance of Combobox
     # @example
-    #   app.checkbox(id: 'id', label: 'label', type: 'YComboBox')
+    #   app.checkbox(id: 'id', label: 'label', class: 'YComboBox')
```

Comment:
existing typo here: s/checkbox/combobox/ in case you you'd like to fix it.

Reply:
makes sense